documentation/operators/src/faq/smoosh-faq.md

```diff
@@ -0,0 +1,88 @@
+# Project Smoosh - FAQ
+
+> We aim on purpose to make minimal changes to reward scheme and software. We're just 'smooshing' together stuff we already debugged and know works.  
+> -- Harry Halpin,  Nym CEO
+
+This page refer to the changes which are planned for Q3 and Q4 2023. As this is a transition period in the beginning (September 2023) the [Mix Nodes FAQ page](./mixnodes-faq.md) holds more answered to the current setup as project Smoosh refers to the eventual setup. As project Smoosh gets progressively implemented the answers on this page will become to be more relevant to the current state and eventually this FAQ page will become to be the main Mix Nodes FAQ page. 
```

Comment:
I think we avoid hard dates?

Reply:
In case we have to change them in the future, more to maintain

Reply:
good point.

Reply:
Shall we address it anyhow as the community keeps asking for that or just drop it.